addons/point_of_sale/static/src/app/models/utils/tax_utils.js

```diff
@@ -57,14 +57,19 @@ export const getTaxesAfterFiscalPosition = (taxes, fiscalPosition, models) => {
 
     const newTaxIds = [];
     for (const tax of taxes) {
-        if (fiscalPosition.tax_mapping_by_ids[tax.id]) {
-            for (const mapTaxId of fiscalPosition.tax_mapping_by_ids[tax.id]) {
-                newTaxIds.push(mapTaxId);
+        const fiscalTaxLines = fiscalPosition.tax_ids.filter(
+            (fiscalTax) => fiscalTax.tax_src_id.id == tax.id
+        );
+        if (fiscalTaxLines.length > 0) {
+            for (const line of fiscalTaxLines) {
+                if (line.tax_dest_id) {
+                    newTaxIds(line.tax_dest_id.id);
```

Comment:
```suggestion
                    newTaxIds.push(line.tax_dest_id.id);
```
Runbot again failed :face_exhaling:

Reply:
@hamo-odoo Did the test fail on this?